README.md

```diff
@@ -219,15 +219,16 @@ See [examples](https://github.com/ClickHouse/clickhouse-rs/tree/main/examples).
 * `lz4` (enabled by default) — enables `Compression::Lz4` and `Compression::Lz4Hc(_)` variants. If enabled, `Compression::Lz4` is used by default for all queries except for `WATCH`.
 * `native-tls` — supports urls with the `HTTPS` schema via `hyper-tls`, which links against OpenSSL.
 * `rustls-tls` — supports urls with the `HTTPS` schema via `hyper-rustls`, which does not link against OpenSSL.
+* `rustls-tls-aws` - use [`aws-lc-rs`](https://github.com/rustls/hyper-rustls?tab=readme-ov-file#crate-features) instead of the default pure Rust `ring` backend for `hyper-rustls`.
 * `inserter` — enables `client.inserter()`.
 * `test-util` — adds mocks. See [the example](https://github.com/ClickHouse/clickhouse-rs/tree/main/examples/mock.rs). Use it only in `dev-dependencies`.
 * `watch` — enables `client.watch` functionality. See the corresponding section for details.
 * `uuid` — adds `serde::uuid` to work with [uuid](https://docs.rs/uuid) crate.
 * `time` — adds `serde::time` to work with [time](https://docs.rs/time) crate.
 
 > **NOTE**:
-> When connecting to ClickHouse via an `HTTPS` url, you must enable either the `native-tls` or `rustls-tls` features.
-> If both are enabled, the `rustls-tls` feature will take precedence.
+> When connecting to ClickHouse via an `HTTPS` url, you must enable either the `native-tls`, `rustls-tls` or `rustls-tls-aws` features.
+> If all are enabled, the priority is `native-tls` > `rustls-tls-aws` > `rustls-tls`.
```

Comment:
The priority is preserved not only if _all_ these features are enabled, but also any of them, so I think it can be rewritten more correctly.